repo/packages/L/linkerd/8/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd",
+  "version": "0.8.3-0.1",
```

Comment:
How are you managing the `version` here? I see that previous Universe version was `0.8.1-0.3`
The format should be linkerdVersion-packagingVersion.